src/sentry/static/sentry/app/views/issueList/filters.tsx

```diff
@@ -2,52 +2,49 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styled from '@emotion/styled';
 
+import {Organization, SavedSearch} from 'app/types';
 import {PageHeader} from 'app/styles/organization';
 import {trackAnalyticsEvent} from 'app/utils/analytics';
 import {t} from 'app/locale';
 import PageHeading from 'app/components/pageHeading';
 import QueryCount from 'app/components/queryCount';
-import SentryTypes from 'app/sentryTypes';
 
 import IssueListSearchBar from './searchBar';
 import IssueListSortOptions from './sortOptions';
 import SavedSearchSelector from './savedSearchSelector';
 
-class IssueListFilters extends React.Component {
-  static propTypes = {
-    organization: SentryTypes.Organization,
-
-    savedSearchList: PropTypes.arrayOf(SentryTypes.SavedSearch),
-    savedSearch: SentryTypes.SavedSearch,
-
-    sort: PropTypes.string,
-    query: PropTypes.string,
-    isSearchDisabled: PropTypes.bool,
-    queryCount: PropTypes.number,
-    queryMaxCount: PropTypes.number,
-
-    onSortChange: PropTypes.func,
-    onSearch: PropTypes.func,
-    onSidebarToggle: PropTypes.func,
-    onSavedSearchSelect: PropTypes.func.isRequired,
-    onSavedSearchDelete: PropTypes.func.isRequired,
-    tagValueLoader: PropTypes.func.isRequired,
-    tags: PropTypes.object.isRequired,
-  };
+const defaultProps = {
+  sort: '',
+  query: null as string | null,
+  onSortChange: (() => {}) as (sort: string) => void,
+  onSearch: (() => {}) as (query: string) => void,
+  onSidebarToggle: (() => {}) as (event: React.MouseEvent) => void,
```

Comment:
Would it be more legible to split this up? 

```
type DefaultProps = {
  ...
};

type Props = {
  ...
} & DefaultProps;

const defaultProps: DefaultProps = {
  ...
};
```

Reply:
Do we actually need the function default props? The component is used in ~one place and should be provided

Reply:
ohh shoot just noticed that. In that case I'll just remove the default props and adjust accordingly